neonvm/controllers/virtualmachine_controller.go

```diff
@@ -503,9 +503,68 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 		// TODO: implement RestartPolicy
 		switch virtualmachine.Spec.RestartPolicy {
 		case vmv1.RestartPolicyAlways:
-			// TODO: restart VirtualMachine
+			log.Info("Restarting VM runner pod", "VM.Phase", virtualmachine.Status.Phase, "RestartPolicy", virtualmachine.Spec.RestartPolicy)
+			// get runner to delete
+			vmRunner := &corev1.Pod{}
+			err := r.Get(ctx, types.NamespacedName{Name: virtualmachine.Status.PodName, Namespace: virtualmachine.Namespace}, vmRunner)
+			if err == nil {
+				// delete current runner
+				if err = r.Delete(ctx, vmRunner); err != nil {
+					log.Error(err, "Failed to delete runner Pod", "Pod.Namespace", vmRunner.Namespace, "Pod.Name", vmRunner.Name)
+					return err
+				}
+				r.Recorder.Event(virtualmachine, "Normal", "Deleted",
+					fmt.Sprintf("VM runner pod was deleted: %s", vmRunner.Name))
```

Comment:
why is this line split? we have a bunch of quite long lines, and I think if we're going to have long lines, we should avoid splits like this except when it improves clarity, and I'm not sure that it makes sense in this case.

Reply:
Recorder Events have the same style over whole code. It was initially generated by kubebuilder and i think it's good.